### PR TITLE
Exit Criteria update for Automations 

### PR DESCRIPTION
**Pull request by Whatthefoxsays**
**Description of the change**:
**Reason for the change**:
**Link to original source**:

Closes #

### CODE REVIEW

File: content/docs/ui/sending-email/getting-started-with-automation.md [hunk 1]
```diff
@@ -297,7 +300,29 @@ You can also find the options to share the signup form in the form editor by cli
...
+*To edit a contact:*
+
+1. Navigate to the Contacts page and search for the contact you want to edit.
+1. Click the contact you wish to edit. This takes you to the contact details page.
+1. To edit any of the profile fields, click on the *Edit* icon to the right of the field.
+1. Once you have finished making all of your changes, click **Save**. 
+
+*To delete a contact from your account:*
+
+1. Navigate to the Contacts page and search for the contact you want to edit.
```
Comment (mollyfriederich) [inline content/docs/ui/sending-email/getting-started-with-automation.md]: My understanding is that based on the 4/10 release the instrux to delete a contact entirely are simpler: 

1. Navigate to the Contacts page and search for the contact you want to edit.
1. Click the delete icon in the Action column to permanently delete the contact from your account. This cannot be undone.

Comment (mollyfriederich) [reply content/docs/ui/sending-email/getting-started-with-automation.md]: @negarzipan

File: content/docs/ui/sending-email/getting-started-with-automation.md [hunk 2]
```diff
@@ -448,11 +473,11 @@ Have feedback or questions? We welcome your thoughts at email_automation@sendgri
   </tr>
   <tr>
     <td>Ability to delete contacts from All Contacts or specific lists.</td>
```
Comment (mollyfriederich) [inline content/docs/ui/sending-email/getting-started-with-automation.md]: "Ability to delete contacts from your account or remove them from specific lists."

@negarzipan

**Comment by thinkingserious** [comment]
Hello @Whatthefoxsays,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**